gatsby/content/projects/2020-01-04-native-gitter-bridge.mdx

```diff
@@ -0,0 +1,37 @@
+---
+layout: project
+title: Native Gitter bridge
+categories:
+  - bridge
+description: Bridges Gitter to the Matrix network
+author: Gitter
+maturity: Released
+language: JavaScript
+license: Apache-2.0
+repo: https://gitlab.com/gitterHQ/webapp
+thumbnail: /docs/projects/images/gitter-logo.svg
+bridges: Gitter
+featured: true
+---
+
+This project bridges [Gitter](https://gitter.im) to Matrix and is already hosted for you. It creates portal rooms for every public room on Gitter which you can join via `#org_repo:gitter.im` (just replace the `/` in the Gitter URI with an underscore `_`). For example https://gitter.im/nodejs/node corresponds to [#nodejs_node:gitter.im]](https://matrix.to/#/#nodejs_node:gitter.im).
+
+If you're already using [Element](https://element.io), you can browse all of the Gitter rooms in the public room directory (switch to the `gitter.im` server).
+
+✓ Public Chats  
+✗ Private Chats  
+✗ 1:1 Chats  
+✓ Native feeling usernames/avatar on both sides (virtual users)
+✓ Replies / Threaded conversations
```

Comment:
This slash is causing formatting issues. Is there a way to workaround this with the slash?

![](https://user-images.githubusercontent.com/558581/103945148-59964b00-50fa-11eb-901b-3f103a48c831.png)

Reply:
Maybe just get rid of the space between like the `usernames/avatars above

Reply:
Fixed in https://github.com/matrix-org/matrix.org/pull/922